### PR TITLE
[Story interactive] Disable overlay on slider when dragging on mobile

### LINKED ISSUE #35260: [Story interactive] prevent education layer display when swiping in slider
Currently the `education layer` displays when a swipe gesture is detected. 
This should not display when the user is interacting with the slider.

Jul-16-2021 09-16-21

### LINKED ISSUE #35261: [Story interactive] prevent player navigation when swiping in slider
### Description

Swiping left or right in the players will navigate to the previous or next story.
This behavior should be suppressed when interacting with the slider.

### Alternatives Considered

-

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by syydneyyhardister**
This disables the drag education while the user is interacting with the slider. 

Demo: https://sydney-story-demo.web.app/examples/amp-story/interactive_slider.html

Closes #35260 
Closes #35261

### CODE REVIEW

**Comment by mszylkowski** [comment]
Since you have the contents of #35340 here, we can wait for that PR to merge before merging this one. LGTM

File: extensions/amp-story-interactive/0.1/amp-story-interactive-slider.js
Comment (processprocess) [inline extensions/amp-story-interactive/0.1/amp-story-interactive-slider.js:117]: Since this is one line, it can be written without brackets like:
`(event) => event.stopPropagation()`

**Review by mszylkowski — APPROVED** [review]

**Review by processprocess — APPROVED** [review]

**Pull request merged**
